steps/oversecured/1.0.0/step.yml

```diff
@@ -0,0 +1,56 @@
+title: Oversecured
+summary: |
+  Detects security vulnerabilities
+description: |
+  A SAST Android vulnerability scanner. Searches for vulnerabilities in [90+ categories](https://oversecured.com/vulnerabilities) using Oversecured API.
+website: https://oversecured.com/
+source_code_url: https://github.com/oversecured/oversecured-bitrise-step
+support_url: https://github.com/oversecured/oversecured-bitrise-step/issues
+published_at: 2020-12-19T23:19:27.761241+03:00
+source:
+  git: https://github.com/oversecured/oversecured-bitrise-step.git
+  commit: d78e7490a88b3a111ec9414c8bb4f8566b26b431
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/oversecured/oversecured-bitrise-step
+deps:
+  brew:
+  - name: git
```

Comment:
It seems that neither git nor wget is needed since step uses go toolkit.

Reply:
@koral-- 
Fixed